src/components/command-bar/components/dialog/header.tsx

```diff
@@ -44,23 +44,30 @@ const CommandBarDialogHeader = () => {
 				</div>
 			) : null}
 			<div className={s.inputWrapper}>
-				<input
-					aria-label={inputPlaceholder}
-					className={s.input}
-					onChange={(e: ChangeEvent<HTMLInputElement>) =>
-						setCurrentInputValue(e.target.value)
-					}
-					placeholder={inputPlaceholder}
-					ref={inputRef}
-					value={currentInputValue}
-				/>
+				{currentCommand.name == 'chat' ? (
+					// This is a dummy element to display text
+					// TODO(kevinwang): make this less hacky
```

Comment:
🎏 Flagging this `TODO` in case it should be resolved before merging!

Reply:
ACK - I'm going to take no action for now